views/books.html

```diff
@@ -0,0 +1,29 @@
+<div class="container">
+  <input class="pull-right" type="search" ng-model="q" placeholder="filter books..." aria-label="filter books" />
+  <form ng-show="add" ng-submit="booksIndexCtrl.createBook(); add = false">
```

Comment:
easuer to understand if you use a different show-hide variable name - ie adding instead of add

"show while (adding)" makes more sense as a state than "show while (add)"